src/common/mod.rs

```diff
@@ -35,6 +35,9 @@ pub enum CoseError {
     DecodeFailed,
     /// CBOR encoding failure.
     EncodeFailed,
+    /// Integer value on wire is outside the expected range
+    /// for the type used to record it
```

Comment:
It might be nice for the comment to make clear that this is a `coset` limitation &ndash; maybe something like:

```rust
/// Integer value on the wire is outside the range of integers representable in this crate.
/// See <https://crates.io/crates/coset/#integer-ranges>.
```

or somesuch.

Reply:
Done and done